soc/arm/nxp_kinetis/ke1xf/Kconfig.series

```diff
@@ -24,5 +24,6 @@ config SOC_SERIES_KINETIS_KE1XF
 	select HAS_MCUX_SCG
 	select HAS_MCUX_WDOG32
 	select HAS_MCUX_FTM
+	select HAS_MCUX_RTC
```

Comment:
it looks like this change is in the wrong file (ke1xf)?

Reply:
@MaureenHelm Thank you for reviewing. Sorry about that. My fingers are so used to typing "ke1x" ;-)

Fixed.